Allure.XUnit/Allure.XUnit.csproj

```diff
@@ -22,7 +22,9 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="Lib.Harmony" Version="2.3.0-prerelease.2" />
```

Comment:
Primarily because of net7.0 support by Harmony (see [here](https://github.com/pardeike/Harmony/releases/tag/v2.3-prerelease.2)). API is the same as in GA version (they only have switched to MonoMod.Core internally). I will create an issue to not to forget to update Harmony as soon as its 2.3 GA is released.